onnxruntime/python/tools/quantization/registry.py

```diff
@@ -60,6 +60,7 @@
     "Concat": QLinearConcat,
     "Softmax": QLinearSoftmax,
     "Where": QLinearWhere,
+    "PRelu": QLinearActivation,
```

Comment:
I believe QLinearActivation would need to be updated to properly handle "PRelu". Onnxruntime doesn't have a QLinearPRelu operator.

Reply:
> I believe QLinearActivation would need to be updated to properly handle "PRelu". Onnxruntime doesn't have a QLinearPRelu operator.

Your reply is very helpful to me. I will try to work on it